core/include/scipp/core/except.h

```diff
@@ -35,7 +35,10 @@ struct SCIPP_CORE_EXPORT TypeError : public std::runtime_error {
   using std::runtime_error::runtime_error;
 
   template <class... Vars>
-  TypeError(const std::string &msg, const Vars &... vars)
+  TypeError(const std::string &msg) : std::runtime_error(msg) {}
```

Comment:
Why is this needed?

Reply:
Some TypeError calls do not have anything in the argpack. I was getting a compile error without this one.